src/test/regress/expected/multi_mx_modifying_xacts.out

```diff
@@ -124,7 +124,7 @@ EXCEPTION
 END $$;
 NOTICE:  caught not_null_violation
 COMMIT;
-ERROR:  could not commit transaction for shard 1220100 on any active node
+ERROR:  was not able to make changes to shard 1220100 on any node
```

Comment:
That part I don't have a problem with ;)  I think I'm otherwise fine with this - I'd like the tests Marco referenced, but if not, then not.